querybook/webapp/components/AIAssistant/AICommandInput.tsx

```diff
@@ -153,21 +242,35 @@ export const AICommandInput: React.FC<AICommandInputProps> = forwardRef(
                         <div className="command">{'/' + command.name}</div>
                     )}
                 </div>
-                <ResizableTextArea
+                <MentionsInput
                     value={commandValue}
                     onChange={handleChange}
-                    className="question-text-area"
+                    onKeyDown={onKeyDown}
+                    style={mentionInputStyle}
                     placeholder={
                         command
                             ? command.hint
-                            : 'Ask AI to generate/edit the query, or type / to see more options'
+                            : 'Ask AI to generate/edit the query. Type @ to select a table. Type / to see more commands. Type Cmd+/ to reset the command.'
```

Comment:
instead of hard coding Cmd+/, can you use KeyMap.aiCommandBar.openCommands instead